virtual_ecosystem/models/plants/__init__.py

```diff
@@ -1,13 +1,27 @@
-"""The :mod:`~virtual_ecosystem.models.plants` module provides a plants model for use
-in the Virtual Ecosystem. The submodules provide:
-
-* The :mod:`~virtual_ecosystem.models.plants.plants_model` submodule provides the
-  PlantsModel class as the main API for interacting with the plants model.
-* The :mod:`~virtual_ecosystem.models.plants.constants` submodule provides dataclasses
-  containing constants used in the model.
-* The :mod:`~virtual_ecosystem.models.plants.community` submodule provides the
-  PlantCohort dataclass that records the details of an individual cohort and the
-  PlantCommunities class that records list of plant cohorts by grid cell.
-"""  # noqa: D205, D415
+"""The :mod:`~virtual_ecosystem.models.plants` module provides
+the :class:`~virtual_ecosystem.models.plants.plants_model.PlantsModel`
+implementation for use in the Virtual Ecosystem.
+
+The main submodule is :mod:`~virtual_ecosystem.models.plants.plants_model` submodule,
+which provides the :class:`~virtual_ecosystem.models.plants.plants_model.PlantsModel`
+class as the main API to initialise and update the plants model.
+
+The other submodules include:
+
+* The :mod:`~virtual_ecosystem.models.plants.constants` submodule provides definitions
+  of the main constants used in the model.
+
+* The :mod:`~virtual_ecosystem.models.plants.functional_types` submodule implements the
+  handling of individual plant functional types and the overall flora definition to be
+  used in a simulation.
+
+* The :mod:`~virtual_ecosystem.models.plants.communities` submodule provides the
+  :class:`~virtual_ecosystem.models.plants.communities.PlantCommunities` class which
+  maps each grid cell on to a representation of the plant community within that cell.
+
```

Comment:
are the cohorts missing here or are they a lower level?

Reply:
That's where it drops into `pyrealm` but I've clarified this.